src/auth-context.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
```

Comment:
Let's move this file in src/providers and rename it to Auth.
As a rule of thumb you should name the filename of a React component in camel case. eg AuthContext.js